graylog2-web-interface/src/components/streamrules/StreamRuleForm.jsx

```diff
@@ -27,11 +27,12 @@ const StreamRuleForm = React.createClass({
     return this.props.streamRule;
   },
   FIELD_PRESENCE_RULE_TYPE: 5,
+  ALWAYS_MATCH_RULE_TYPE: 7,
   _resetValues() {
     this.setState(this.props.streamRule);
   },
   _onSubmit() {
-    if (this.state.type === this.FIELD_PRESENCE_RULE_TYPE) {
+    if (this.state.type === this.FIELD_PRESENCE_RULE_TYPE || this.state.type === this.ALWAYS_MATCH_RULE_TYPE) {
```

Comment:
Please update the code so that there is no "value" form field when the always match rule is selected. (similar how it's done with the field presence rule)